fdbserver/Resolver.actor.cpp

```diff
@@ -675,9 +680,24 @@ ACTOR Future<Void> resolverCore(ResolverInterface resolver,
 		}
 	}
 
+	state std::unordered_map<EncryptCipherDomainId, Reference<BlobCipherKey>> cipherKeys;
```

Comment:
We need to fetch latest cipher keys for each resolver request. The cipher keys fetched here could be staled and need refresh. Preferably the cipher key fetch request could overlap with some other async operations.

Reply:
Good point i can do it inside the actual calls